### ISSUE TITLE
Highlighter crashes CSharpRepl when editing a class with a property that has an unknown attribute

### ISSUE THREAD

**Issue by RobotsOnDrugs** [waf/CSharpRepl#227]
### Version

0.5.1

### What happened?

If you paste a class like this and go to edit a highlighted word (such as "record" here), the program crashes. I have not tested it much so I am not sure, but it very much appears to be related to the fact that there is an undefined attribute. If you simply hit enter to define this class, there is only a compiler error as expected.
```cs
internal record Options
{
	[Option(HelpText = "Time in milliseconds to show the notification.")]
	public int NotificationTime { get; init; } = 60 * 1000;
}
```
Exception stack trace:
```cs
Unhandled exception. System.NullReferenceException: Object reference not set to an instance of an object.
   at Microsoft.CodeAnalysis.CSharp.SyntaxTreeSemanticModel.<CreateModelForAttribute>g__getAttributeTarget|79_0(SyntaxNode targetSyntax)
   at Microsoft.CodeAnalysis.CSharp.SyntaxTreeSemanticModel.CreateModelForAttribute(Binder enclosingBinder, AttributeSyntax attribute, MemberSemanticModel containingModel)
   at Microsoft.CodeAnalysis.CSharp.SyntaxTreeSemanticModel.CreateMemberModel(CSharpSyntaxNode node)
   at Microsoft.CodeAnalysis.CSharp.SyntaxTreeSemanticModel.GetOrAddModel(CSharpSyntaxNode node)
   at Microsoft.CodeAnalysis.CSharp.SyntaxTreeSemanticModel.GetOrAddModelForAttribute(AttributeSyntax attribute)
   at Microsoft.CodeAnalysis.CSharp.SyntaxTreeSemanticModel.GetMemberModel(SyntaxNode node)
   at Microsoft.CodeAnalysis.CSharp.SyntaxTreeSemanticModel.GetSymbolInfoWorker(CSharpSyntaxNode node, SymbolInfoOptions options, CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.CSharp.CSharpSemanticModel.GetSymbolInfo(ExpressionSyntax expression, CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.CSharp.CSharpExtensions.GetSymbolInfo(SemanticModel semanticModel, ExpressionSyntax expression, CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.CSharp.Classification.Classifiers.NameSyntaxClassifier.ClassifyTypeSyntax(NameSyntax name, SemanticModel semanticModel, ArrayBuilder`1 result, CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.CSharp.Classification.Classifiers.NameSyntaxClassifier.AddClassifications(SyntaxNode syntax, SemanticModel semanticModel, ClassificationOptions options, ArrayBuilder`1 result, CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.Classification.AbstractSyntaxClassificationService.Worker.ClassifyNode(SyntaxNode syntax)
   at Microsoft.CodeAnalysis.Classification.AbstractSyntaxClassificationService.Worker.ClassifyNodeOrToken(SyntaxNodeOrToken nodeOrToken)
   at Microsoft.CodeAnalysis.Classification.AbstractSyntaxClassificationService.Worker.ProcessNodes()
   at Microsoft.CodeAnalysis.Classification.AbstractSyntaxClassificationService.Worker.Classify(SemanticModel semanticModel, TextSpan textSpan, ArrayBuilder`1 list, Func`2 getNodeClassifiers, Func`2 getTokenClassifiers, ClassificationOptions options, CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.Classification.AbstractSyntaxClassificationService.AddSemanticClassifications(SemanticModel semanticModel, TextSpan textSpan, Func`2 getNodeClassifiers, Func`2 getTokenClassifiers, ArrayBuilder`1 result, ClassificationOptions options, CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.Classification.Classifier.GetClassifiedSpans(HostWorkspaceServices workspaceServices, Project project, SemanticModel semanticModel, TextSpan textSpan, ClassificationOptions options, CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.Classification.Classifier.GetClassifiedSpansAsync(Document document, TextSpan textSpan, CancellationToken cancellationToken)
   at CSharpRepl.Services.SyntaxHighlighting.SyntaxHighlighter.HighlightAsync(Document document) in D:\a\CSharpRepl\CSharpRepl\CSharpRepl.Services\SyntaxHighlighting\SyntaxHighlighter.cs:line 43
   at CSharpRepl.Services.Roslyn.RoslynServices.SyntaxHighlightAsync(String text) in D:\a\CSharpRepl\CSharpRepl\CSharpRepl.Services\Roslyn\RoslynServices.cs:line 153
   at CSharpRepl.PrettyPromptConfig.CSharpReplPromptCallbacks.HighlightCallbackAsync(String text, CancellationToken cancellationToken) in D:\a\CSharpRepl\CSharpRepl\CSharpRepl\CSharpReplPromptCallbacks.cs:line 117
   at PrettyPrompt.Highlighting.SyntaxHighlighter.HighlightAsync(String input, CancellationToken cancellationToken)
   at PrettyPrompt.Prompt.ReadLineAsync()
   at CSharpRepl.ReadEvalPrintLoop.RunAsync(Configuration config) in D:\a\CSharpRepl\CSharpRepl\CSharpRepl\ReadEvalPrintLoop.cs:line 46
   at CSharpRepl.Program.Main(String[] args) in D:\a\CSharpRepl\CSharpRepl\CSharpRepl\Program.cs:line 52
   at CSharpRepl.Program.Main(String[] args) in D:\a\CSharpRepl\CSharpRepl\CSharpRepl\Program.cs:line 58
   at CSharpRepl.Program.<Main>(String[] args)
```

Trace log:
```
2023-02-18T05:55:44 - Trace session starting
2023-02-18T05:55:44 - Starting background initialization
2023-02-18T05:55:44 - Available Reference Assemblies: "C:\Program Files\dotnet\packs\Microsoft.NETCore.App.Ref": ["3.1.0", "5.0.0", "6.0.0-preview.7.21377.19", "6.0.0-rc.1.21451.13", "6.0.10", "6.0.12", "6.0.14", "7.0.0", "7.0.3"]
2023-02-18T05:55:44 - Available Implementation Assemblies: "C:\Program Files\dotnet\shared\Microsoft.NETCore.App": ["3.0.3", "3.1.32", "5.0.14", "5.0.17", "5.0.8", "6.0.0-preview.7.21377.19", "6.0.0-rc.1.21451.13", "6.0.10", "6.0.12", "6.0.14", "7.0.0", "7.0.3"]
2023-02-18T05:55:44 - .NET Version: Microsoft.NETCore.App / 7.0.3
2023-02-18T05:55:44 - Reference Assembly Paths: C:\Program Files\dotnet\packs\Microsoft.NETCore.App.Ref\7.0.3\ref\net7.0
2023-02-18T05:55:44 - Implementation Assembly Paths: C:\Program Files\dotnet\shared\Microsoft.NETCore.App\7.0.3
2023-02-18T05:55:44 - Shared Framework Paths: C:\Program Files\dotnet\shared\Microsoft.NETCore.App\7.0.3
2023-02-18T05:55:44 - Loaded Reference Assemblies: "C:\Program Files\dotnet\packs\Microsoft.NETCore.App.Ref\7.0.3\ref\net7.0": ["System.Net.WebSockets.dll", "System.Runtime.Handles.dll", "System.Security.Cryptography.Primitives.dll", "System.Console.dll", "System.Globalization.Calendars.dll", "System.Linq.Queryable.dll", "System.Numerics.dll", "System.Runtime.InteropServices.dll", "System.Security.Cryptography.X509Certificates.dll", "System.Core.dll", "System.Globalization.dll", "System.Memory.dll", "System.Numerics.Vectors.dll", "System.Runtime.InteropServices.JavaScript.dll", "System.Security.dll", "System.Data.Common.dll", "System.Globalization.Extensions.dll", "System.Net.dll", "System.ObjectModel.dll", "System.Runtime.InteropServices.RuntimeInformation.dll", "System.Security.Principal.dll", "System.Data.DataSetExtensions.dll", "System.IO.Compression.Brotli.dll", "System.Net.Http.dll", "System.Reflection.DispatchProxy.dll", "System.Runtime.Intrinsics.dll", "System.Security.Principal.Windows.dll", "System.Data.dll", "System.IO.Compression.dll", "System.Net.Http.Json.dll", "System.Reflection.dll", "System.Runtime.Loader.dll", "System.Security.SecureString.dll", "System.Diagnostics.Contracts.dll", "System.IO.Compression.FileSystem.dll", "System.Net.HttpListener.dll", "System.Reflection.Emit.dll", "System.Runtime.Numerics.dll", "System.ServiceModel.Web.dll", "System.Diagnostics.Debug.dll", "System.IO.Compression.ZipFile.dll", "System.Net.Mail.dll", "System.Reflection.Emit.ILGeneration.dll", "System.Runtime.Serialization.dll", "System.ServiceProcess.dll", "System.Diagnostics.DiagnosticSource.dll", "System.IO.dll", "System.Net.NameResolution.dll", "System.Reflection.Emit.Lightweight.dll", "System.Runtime.Serialization.Formatters.dll", "System.Text.Encoding.CodePages.dll", "System.Diagnostics.FileVersionInfo.dll", "System.IO.FileSystem.AccessControl.dll", "System.Net.NetworkInformation.dll", "System.Reflection.Extensions.dll", "System.Runtime.Serialization.Json.dll", "System.Text.Encoding.dll", "System.Diagnostics.Process.dll", "System.IO.FileSystem.dll", "System.Net.Ping.dll", "System.Reflection.Metadata.dll", "System.Runtime.Serialization.Primitives.dll", "System.Text.Encoding.Extensions.dll", "System.Diagnostics.StackTrace.dll", "System.IO.FileSystem.DriveInfo.dll", "System.Net.Primitives.dll", "System.Reflection.Primitives.dll", "System.Runtime.Serialization.Xml.dll", "System.Text.Encodings.Web.dll", "System.Diagnostics.TextWriterTraceListener.dll", "System.IO.FileSystem.Primitives.dll", "System.Net.Quic.dll", "System.Reflection.TypeExtensions.dll", "System.Security.AccessControl.dll", "System.Text.Json.dll", "System.Diagnostics.Tools.dll", "System.IO.FileSystem.Watcher.dll", "System.Net.Requests.dll", "System.Resources.Reader.dll", "System.Security.Claims.dll", "System.Text.RegularExpressions.dll", "System.Diagnostics.TraceSource.dll", "System.IO.IsolatedStorage.dll", "System.Net.Security.dll", "System.Resources.ResourceManager.dll", "System.Security.Cryptography.Algorithms.dll", "System.Threading.Channels.dll", "System.Diagnostics.Tracing.dll", "System.IO.MemoryMappedFiles.dll", "System.Net.ServicePoint.dll", "System.Resources.Writer.dll", "System.Security.Cryptography.Cng.dll", "System.Threading.dll", "System.dll", "System.IO.Pipes.AccessControl.dll", "System.Net.Sockets.dll", "System.Runtime.CompilerServices.Unsafe.dll", "System.Security.Cryptography.Csp.dll", "System.Threading.Overlapped.dll", "System.Drawing.dll", "System.IO.Pipes.dll", "System.Net.WebClient.dll", "System.Runtime.CompilerServices.VisualC.dll", "System.Security.Cryptography.dll", "System.Threading.Tasks.Dataflow.dll", "System.Drawing.Primitives.dll", "System.IO.UnmanagedMemoryStream.dll", "System.Net.WebHeaderCollection.dll", "System.Runtime.dll", "System.Security.Cryptography.Encoding.dll", "System.Threading.Tasks.dll", "System.Dynamic.Runtime.dll", "System.Linq.dll", "System.Net.WebProxy.dll", "System.Runtime.Extensions.dll", "System.Security.Cryptography.OpenSsl.dll", "System.Threading.Tasks.Extensions.dll", "System.Formats.Asn1.dll", "System.Linq.Expressions.dll", "System.Net.WebSockets.Client.dll", "System.Formats.Tar.dll", "System.Linq.Parallel.dll", "Microsoft.CSharp.dll", "Microsoft.VisualBasic.Core.dll", "Microsoft.VisualBasic.dll", "Microsoft.Win32.Primitives.dll", "Microsoft.Win32.Registry.dll", "mscorlib.dll", "netstandard.dll", "System.AppContext.dll", "System.Buffers.dll", "System.Collections.Concurrent.dll", "System.Collections.dll", "System.Collections.Immutable.dll", "System.Collections.NonGeneric.dll", "System.Collections.Specialized.dll", "System.ComponentModel.Annotations.dll", "System.ComponentModel.DataAnnotations.dll", "System.ComponentModel.dll", "System.ComponentModel.EventBasedAsync.dll", "System.ComponentModel.Primitives.dll", "System.ComponentModel.TypeConverter.dll", "System.Configuration.dll", "System.Threading.Tasks.Parallel.dll", "System.Threading.Thread.dll", "System.Threading.ThreadPool.dll", "System.Threading.Timer.dll", "System.Transactions.dll", "System.Transactions.Local.dll", "System.ValueTuple.dll", "System.Web.dll", "System.Web.HttpUtility.dll", "System.Windows.dll", "System.Xml.dll", "System.Xml.Linq.dll", "System.Xml.ReaderWriter.dll", "System.Xml.Serialization.dll", "System.Xml.XDocument.dll", "System.Xml.XmlDocument.dll", "System.Xml.XmlSerializer.dll", "System.Xml.XPath.dll", "System.Xml.XPath.XDocument.dll", "WindowsBase.dll"]
2023-02-18T05:55:44 - Loaded Implementation Assemblies: "C:\Program Files\dotnet\shared\Microsoft.NETCore.App\7.0.3": ["System.Resources.ResourceManager.dll", "System.Resources.Writer.dll", "System.Runtime.CompilerServices.Unsafe.dll", "System.Runtime.CompilerServices.VisualC.dll", "System.Runtime.dll", "System.Runtime.Extensions.dll", "System.Runtime.Handles.dll", "System.Runtime.InteropServices.dll", "System.Runtime.InteropServices.JavaScript.dll", "System.Runtime.InteropServices.RuntimeInformation.dll", "System.Runtime.Intrinsics.dll", "System.Runtime.Loader.dll", "System.Runtime.Numerics.dll", "System.Runtime.Serialization.dll", "System.Runtime.Serialization.Formatters.dll", "System.Runtime.Serialization.Json.dll", "System.Runtime.Serialization.Primitives.dll", "System.Runtime.Serialization.Xml.dll", "System.Security.AccessControl.dll", "System.Security.Claims.dll", "System.Security.Cryptography.Algorithms.dll", "System.Security.Cryptography.Cng.dll", "System.Threading.Tasks.dll", "System.Threading.Tasks.Extensions.dll", "System.Threading.Tasks.Parallel.dll", "System.Threading.Thread.dll", "System.Threading.ThreadPool.dll", "System.Threading.Timer.dll", "System.Transactions.dll", "System.Transactions.Local.dll", "System.ValueTuple.dll", "System.Web.dll", "System.Web.HttpUtility.dll", "System.Windows.dll", "System.Xml.dll", "System.Xml.Linq.dll", "System.Xml.ReaderWriter.dll", "System.Xml.Serialization.dll", "System.Xml.XDocument.dll", "System.Xml.XmlDocument.dll", "System.Xml.XmlSerializer.dll", "System.Xml.XPath.dll", "System.Xml.XPath.XDocument.dll", "WindowsBase.dll", "System.Security.Cryptography.Csp.dll", "System.Security.Cryptography.dll", "System.Security.Cryptography.Encoding.dll", "System.Security.Cryptography.OpenSsl.dll", "System.Security.Cryptography.Primitives.dll", "System.Security.Cryptography.X509Certificates.dll", "System.Security.dll", "System.Security.Principal.dll", "System.Security.Principal.Windows.dll", "System.Security.SecureString.dll", "System.ServiceModel.Web.dll", "System.ServiceProcess.dll", "System.Text.Encoding.CodePages.dll", "System.Text.Encoding.dll", "System.Text.Encoding.Extensions.dll", "System.Text.Encodings.Web.dll", "System.Text.Json.dll", "System.Text.RegularExpressions.dll", "System.Threading.Channels.dll", "System.Threading.dll", "System.Threading.Overlapped.dll", "System.Threading.Tasks.Dataflow.dll", "System.Collections.NonGeneric.dll", "System.Collections.Specialized.dll", "System.ComponentModel.Annotations.dll", "System.ComponentModel.DataAnnotations.dll", "System.ComponentModel.dll", "System.ComponentModel.EventBasedAsync.dll", "System.ComponentModel.Primitives.dll", "System.ComponentModel.TypeConverter.dll", "System.Configuration.dll", "System.Console.dll", "System.Core.dll", "System.Data.Common.dll", "System.Data.DataSetExtensions.dll", "System.Data.dll", "System.Diagnostics.Contracts.dll", "System.Diagnostics.Debug.dll", "System.Diagnostics.DiagnosticSource.dll", "System.Diagnostics.FileVersionInfo.dll", "System.Diagnostics.Process.dll", "System.Diagnostics.StackTrace.dll", "System.Diagnostics.TextWriterTraceListener.dll", "System.Diagnostics.Tools.dll", "System.Diagnostics.TraceSource.dll", "System.IO.Pipes.AccessControl.dll", "System.IO.Pipes.dll", "System.IO.UnmanagedMemoryStream.dll", "System.Linq.dll", "System.Linq.Expressions.dll", "System.Linq.Parallel.dll", "System.Linq.Queryable.dll", "System.Memory.dll", "System.Net.dll", "System.Net.Http.dll", "System.Net.Http.Json.dll", "System.Net.HttpListener.dll", "System.Net.Mail.dll", "System.Net.NameResolution.dll", "System.Net.NetworkInformation.dll", "System.Net.Ping.dll", "System.Net.Primitives.dll", "System.Net.Quic.dll", "System.Net.Requests.dll", "System.Net.Security.dll", "System.Net.ServicePoint.dll", "System.Net.Sockets.dll", "System.Net.WebClient.dll", "Microsoft.CSharp.dll", "Microsoft.VisualBasic.Core.dll", "Microsoft.VisualBasic.dll", "Microsoft.Win32.Primitives.dll", "Microsoft.Win32.Registry.dll", "mscorlib.dll", "netstandard.dll", "System.AppContext.dll", "System.Buffers.dll", "System.Collections.Concurrent.dll", "System.Collections.dll", "System.Collections.Immutable.dll", "System.Net.WebHeaderCollection.dll", "System.Net.WebProxy.dll", "System.Net.WebSockets.Client.dll", "System.Net.WebSockets.dll", "System.Numerics.dll", "System.Numerics.Vectors.dll", "System.ObjectModel.dll", "System.Private.CoreLib.dll", "System.Private.DataContractSerialization.dll", "System.Private.Uri.dll", "System.Private.Xml.dll", "System.Private.Xml.Linq.dll", "System.Reflection.DispatchProxy.dll", "System.Reflection.dll", "System.Reflection.Emit.dll", "System.Reflection.Emit.ILGeneration.dll", "System.Reflection.Emit.Lightweight.dll", "System.Reflection.Extensions.dll", "System.Reflection.Metadata.dll", "System.Reflection.Primitives.dll", "System.Reflection.TypeExtensions.dll", "System.Resources.Reader.dll", "System.Diagnostics.Tracing.dll", "System.dll", "System.Drawing.dll", "System.Drawing.Primitives.dll", "System.Dynamic.Runtime.dll", "System.Formats.Asn1.dll", "System.Formats.Tar.dll", "System.Globalization.Calendars.dll", "System.Globalization.dll", "System.Globalization.Extensions.dll", "System.IO.Compression.Brotli.dll", "System.IO.Compression.dll", "System.IO.Compression.FileSystem.dll", "System.IO.Compression.ZipFile.dll", "System.IO.dll", "System.IO.FileSystem.AccessControl.dll", "System.IO.FileSystem.dll", "System.IO.FileSystem.DriveInfo.dll", "System.IO.FileSystem.Primitives.dll", "System.IO.FileSystem.Watcher.dll", "System.IO.IsolatedStorage.dll", "System.IO.MemoryMappedFiles.dll"]
2023-02-18T05:55:45 - MEF Default Assemblies: C:\Users\iamtheclaw\.dotnet\tools\.store\csharprepl\0.5.1\csharprepl\0.5.1\tools\net7.0\any\Microsoft.CodeAnalysis.Workspaces.dll, C:\Users\iamtheclaw\.dotnet\tools\.store\csharprepl\0.5.1\csharprepl\0.5.1\tools\net7.0\any\Microsoft.CodeAnalysis.CSharp.Workspaces.dll, C:\Users\iamtheclaw\.dotnet\tools\.store\csharprepl\0.5.1\csharprepl\0.5.1\tools\net7.0\any\Microsoft.CodeAnalysis.Features.dll, C:\Users\iamtheclaw\.dotnet\tools\.store\csharprepl\0.5.1\csharprepl\0.5.1\tools\net7.0\any\Microsoft.CodeAnalysis.CSharp.Features.dll
2023-02-18T05:55:45 - Background initialization complete
2023-02-18T05:55:45 - Warm-up Starting
2023-02-18T05:55:48 - Warm-up Complete
2023-02-18T05:55:59 - Unhandled Exception: System.NullReferenceException: Object reference not set to an instance of an object.
   at Microsoft.CodeAnalysis.CSharp.SyntaxTreeSemanticModel.<CreateModelForAttribute>g__getAttributeTarget|79_0(SyntaxNode targetSyntax)
   at Microsoft.CodeAnalysis.CSharp.SyntaxTreeSemanticModel.CreateModelForAttribute(Binder enclosingBinder, AttributeSyntax attribute, MemberSemanticModel containingModel)
   at Microsoft.CodeAnalysis.CSharp.SyntaxTreeSemanticModel.CreateMemberModel(CSharpSyntaxNode node)
   at Microsoft.CodeAnalysis.CSharp.SyntaxTreeSemanticModel.GetOrAddModel(CSharpSyntaxNode node)
   at Microsoft.CodeAnalysis.CSharp.SyntaxTreeSemanticModel.GetOrAddModelForAttribute(AttributeSyntax attribute)
   at Microsoft.CodeAnalysis.CSharp.SyntaxTreeSemanticModel.GetMemberModel(SyntaxNode node)
   at Microsoft.CodeAnalysis.CSharp.SyntaxTreeSemanticModel.GetSymbolInfoWorker(CSharpSyntaxNode node, SymbolInfoOptions options, CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.CSharp.CSharpSemanticModel.GetSymbolInfo(ExpressionSyntax expression, CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.CSharp.CSharpExtensions.GetSymbolInfo(SemanticModel semanticModel, ExpressionSyntax expression, CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.CSharp.Classification.Classifiers.NameSyntaxClassifier.ClassifyTypeSyntax(NameSyntax name, SemanticModel semanticModel, ArrayBuilder`1 result, CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.CSharp.Classification.Classifiers.NameSyntaxClassifier.AddClassifications(SyntaxNode syntax, SemanticModel semanticModel, ClassificationOptions options, ArrayBuilder`1 result, CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.Classification.AbstractSyntaxClassificationService.Worker.ClassifyNode(SyntaxNode syntax)
   at Microsoft.CodeAnalysis.Classification.AbstractSyntaxClassificationService.Worker.ClassifyNodeOrToken(SyntaxNodeOrToken nodeOrToken)
   at Microsoft.CodeAnalysis.Classification.AbstractSyntaxClassificationService.Worker.ProcessNodes()
   at Microsoft.CodeAnalysis.Classification.AbstractSyntaxClassificationService.Worker.Classify(SemanticModel semanticModel, TextSpan textSpan, ArrayBuilder`1 list, Func`2 getNodeClassifiers, Func`2 getTokenClassifiers, ClassificationOptions options, CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.Classification.AbstractSyntaxClassificationService.AddSemanticClassifications(SemanticModel semanticModel, TextSpan textSpan, Func`2 getNodeClassifiers, Func`2 getTokenClassifiers, ArrayBuilder`1 result, ClassificationOptions options, CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.Classification.Classifier.GetClassifiedSpans(HostWorkspaceServices workspaceServices, Project project, SemanticModel semanticModel, TextSpan textSpan, ClassificationOptions options, CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.Classification.Classifier.GetClassifiedSpansAsync(Document document, TextSpan textSpan, CancellationToken cancellationToken)
   at CSharpRepl.Services.SyntaxHighlighting.SyntaxHighlighter.HighlightAsync(Document document) in D:\a\CSharpRepl\CSharpRepl\CSharpRepl.Services\SyntaxHighlighting\SyntaxHighlighter.cs:line 43
   at CSharpRepl.Services.Roslyn.RoslynServices.SyntaxHighlightAsync(String text) in D:\a\CSharpRepl\CSharpRepl\CSharpRepl.Services\Roslyn\RoslynServices.cs:line 153
   at CSharpRepl.PrettyPromptConfig.CSharpReplPromptCallbacks.HighlightCallbackAsync(String text, CancellationToken cancellationToken) in D:\a\CSharpRepl\CSharpRepl\CSharpRepl\CSharpReplPromptCallbacks.cs:line 117
   at PrettyPrompt.Highlighting.SyntaxHighlighter.HighlightAsync(String input, CancellationToken cancellationToken)
   at PrettyPrompt.Prompt.ReadLineAsync()
   at CSharpRepl.ReadEvalPrintLoop.RunAsync(Configuration config) in D:\a\CSharpRepl\CSharpRepl\CSharpRepl\ReadEvalPrintLoop.cs:line 46
   at CSharpRepl.Program.Main(String[] args) in D:\a\CSharpRepl\CSharpRepl\CSharpRepl\Program.cs:line 52
   at CSharpRepl.Program.Main(String[] args) in D:\a\CSharpRepl\CSharpRepl\CSharpRepl\Program.cs:line 58
   at CSharpRepl.Program.<Main>(String[] args)
```

**Comment by waf** [comment]
Thanks for opening this! I'm able to reproduce this, but not in the latest `main` branch. I'll do a bit more research and see if I can track down what's responsible for fixing this. Regardless, I think we're due for a 0.5.2 release.

**Comment by waf** [comment]
Finally got around to sending out a release. This is fixed in 0.6.0. Thanks for reporting.
